classes/form/cms_types_form.php

```diff
@@ -59,6 +59,11 @@ protected function definition() {
         $mform->addElement('editor', 'description', get_string('description'));
         $mform->setType('description', PARAM_RAW);
 
+        $select = $mform->addElement('select', 'datasources', get_string('datasources', 'mod_cms'),
```

Comment:
can you please convert this to a checkbox group, multi selects are too easy to deselect the thing you want to keep when turning something else on or off

Reply:
done, although no 'select all/deselect all'. Couldn't get it to work.